azure-pipelines.yml

```diff
@@ -16,18 +16,19 @@ jobs:
         docker run --rm --privileged docker.io/tonistiigi/binfmt:latest --install all
         python -m pip install dependency-groups
         python -m dependency_groups test | xargs python -m pip install -e.
-        python ./bin/run_tests.py
+        python ./bin/run_tests.py --num-processes 2
```

Comment:
```suggestion
        python ./bin/run_tests.py
```

Reply:
Afaiks these and your other suggestions are in your commit, did I miss something?

Reply:
Yes, I rebased and removed this now that the fix in in main.